Assignment2/Q1.java

```diff
@@ -0,0 +1,148 @@
+//The tree created is a binary tree to match the example given but the method does not make any use of binary tree features 
+
+class Q1
+{
+	public static void main(String[] args)
+	{
+		BinaryTree tree = new BinaryTree(16);
+		tree.add(9);
+		tree.add(18);
+		tree.add(19);
+		tree.add(3);
+		tree.add(14);
+		tree.add(1);
+		tree.add(5);
+
+		System.out.println("OK");
+
+		find(tree, 5);
+	}
+
+	public static void find(BinaryTree tree, int valueToFind)
+	{
+		if (tree.root.value == valueToFind)
+		{
+			System.out.println(valueToFind + " is the root - it has no ancestors");
+		}
+		else if(findHelp(tree.root, valueToFind) == 0)
+		{
+			System.out.println(valueToFind + " is not part of the tree");
+		}
+	}
+
+	public static int findHelp(Node consider, int valueToFind)
```

Comment:
Looking back it should be boolean - I was initially doing a different method where I was using different integers to show different outcomes

Yes, I will change it to private - initially this was my main method and then I changed it to the helper method